lib/cocoapods/installer.rb

```diff
@@ -43,6 +43,7 @@ class Installer
     autoload :SandboxHeaderPathsInstaller,  'cocoapods/installer/sandbox_header_paths_installer'
     autoload :SandboxDirCleaner,            'cocoapods/installer/sandbox_dir_cleaner'
     autoload :ProjectCache,                 'cocoapods/installer/project_cache/project_cache'
+    autoload :TargetUUIDGenerator,          'cocoapods/installer/xcode/target_uuid_generator'
```

Comment:
this auto load should be nested somewhere in the `Xcode` namespace

Reply:
I've noticed that the source file for `TargetUUIDGenerator` isn't correctly placed, I moved it now.